src/esp/bindings/SensorBindings.cpp

```diff
@@ -112,7 +112,26 @@ void initSensorBindings(py::module& m) {
              VisualSensor, Magnum::SceneGraph::PyFeatureHolder<CameraSensor>>(
       m, "CameraSensor")
       .def(py::init_alias<std::reference_wrapper<scene::SceneNode>,
-                          const SensorSpec::ptr&>());
+                          const SensorSpec::ptr&>())
+      .def("set_projection_params", &CameraSensor::setProjectionParameters,
+           R"(Specify the projection parameters this CameraSensor should use.
+           Should be consumed by first querying this CameraSensor's SensorSpec
+           and then modifying as necessary.)",
+           "sensor_spec"_a)
+      .def(
+          "set_fov",
+          static_cast<void (CameraSensor::*)(Mn::Deg)>(&CameraSensor::setFOV),
+          R"(Set the field of view to use for this CameraSensor.  Only applicable to 
+          Pinhole Camera Types)",
+          "fov"_a)
+      .def(
```

Comment:
Why isn't this a property too?

Reply:
because it is a modifier to a zoom scaling matrix, not a property.

Reply:
camera_model.zoom_scale_matrix ?

Reply:
Why would i expose this though? It is arcane and could easily mess up the projection if not maintained properly.

Reply:
Then don't other exposing it.

Reply:
Would python users really want to access this?

Reply:
My point being, if you don't think they would, then don't bother exposing it.

I feel like they definitely would want to access the FOV though.

Reply:
My rationale for 'restricting' access to that was that it only had meaning when used with a pinhole camera, so i didn't want it to seem like a "first class citizen", so to speak.   I am fine with making it a property if you think that would be appropriate. :).

Reply:
You could always make it throw an error if modified / accessed for an ortho, no?

Reply:
How about just an informational message?  An error seems drastic, but I will defer to your greater experience as a user of the system :).

Reply:
Works for me.